collector/pg_stat_activity_summary.go

```diff
@@ -0,0 +1,152 @@
+// Copyright 2023 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package collector
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/go-kit/log"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const statActivitySummarySubsystem = "stat_activity_summary"
+
+func init() {
+	registerCollector(statActivitySummarySubsystem, defaultEnabled, NewPGStatActivitySummaryCollector)
+}
+
+type PGStatActivitySummaryCollector struct {
+	log log.Logger
+}
+
+func NewPGStatActivitySummaryCollector(config collectorConfig) (Collector, error) {
+	return &PGStatActivitySummaryCollector{log: config.logger}, nil
+}
+
+var (
+	statActivitySummaryActiveCount = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, statActivitySummarySubsystem, "active_count"),
```

Comment:
`_count` is reserved for Histogram counters.

```suggestion
		prometheus.BuildFQName(namespace, statActivitySummarySubsystem, "active"),
```